app/javascript/controllers/map_controller.js

```diff
@@ -252,9 +252,6 @@ export default class extends Controller {
         el.classList.add("govuk-!-display-none")
       }
     }
-
-    const heading = this.element.querySelector("h3")
-    heading.innerHTML = latlng === null ? "All neighbours" : "Neighbours"
```

Comment:
Does this need to be removed?

Reply:
Design feedback - shows up in the constraints list too